pkg/controller/machineset.go

```diff
@@ -425,11 +425,6 @@ func (c *controller) manageReplicas(allMachines []*v1alpha1.Machine, machineSet
 // meaning it did not expect to see any more of its machines created or deleted. This function is not meant to be
 // invoked concurrently with the same key.
 func (c *controller) reconcileClusterMachineSet(key string) error {
-	startTime := time.Now()
-	defer func() {
```

Comment:
No harm in having it ryt ?

Reply:
Restored it with printing at start and end of reconcile.